src/qp_klp/WorkflowFactory.py

```diff
@@ -19,6 +19,7 @@ class WorkflowFactory():
     ST_TO_IN_MAP = {PROTOCOL_NAME_ILLUMINA: ['standard_metag',
                                              'standard_metat',
                                              'absquant_metag',
+                                             'abs_quant_metag',
```

Comment:
Given that this module already depends on `metapool`, I would really, **really** like it if, instead of redefining strings for the same sheet-type names that are (canonically) defined in `metapool.sample_sheet`, we just used the public values defined there:

https://github.com/biocore/kl-metapool/blob/7bb932f1cb8a69e49c087710e14de046eaef0079/metapool/sample_sheet.py#L43-L47

Also, there still is no `absquant_metat` (and there are no plans that there ever will be).  Using (only) the sheet types from in the module where we actually define the sheet types would ensure that we never have this sort of nonsense entry in the future.